<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JanelaDeLogin</class>
 <widget class="QDialog" name="JanelaDeLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>207</width>
     <height>94</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="labelSenha">
        <property name="styleSheet">
         <string notr="true">color: #00BFFF;
</string>
        </property>
        <property name="text">
         <string>Senha:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelUsuario">
        <property name="styleSheet">
         <string notr="true">color: #00BFFF;
</string>
        </property>
        <property name="text">
         <string>Usuario:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditSenha">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditUsuario">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButtonEntrar">
        <property name="styleSheet">
         <string notr="true">color: #00BFFF;
background-color: #D3D3D3;</string>
        </property>
        <property name="text">
         <string>Entrar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="logoEmpresa">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>391</width>
     <height>351</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>slogan.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>logoEmpresa</zorder>
  <zorder>layoutWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
